docs/docker-compose/config/server.cfg

```diff
@@ -0,0 +1,11 @@
+enable_sso = yes
+# (external) address of this Kopano system
+kcoidc_issuer_identifier = https://kopano.demo:9443
```

Comment:
I don't understand why u use port 9443 everywhere (only have it listen on port 9443, but expose it on 443 to the exsternal world together with the external hostname).

Reply:
That is mostly because I have already something running on port 443 on my local system.